x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IndexLifecycleRunner.java

```diff
@@ -219,7 +263,7 @@ public void runPolicyAfterStateChange(String policy, IndexMetaData indexMetaData
             logger.debug("policy [{}] for index [{}] complete, skipping execution", policy, index);
             return;
         } else if (currentStep instanceof ErrorStep) {
-            logger.debug("policy [{}] for index [{}] on an error step, skipping execution", policy, index);
+            onErrorMaybeRetryFailedStep(policy, indexMetaData);
```

Comment:
I actually think we should go back to just skipping it here. We can leave automatic ILM retries happening *only* on the periodic ILM interval (every 10 minutes by default) rather than every time state changes. This will work for our "backoff" of automatic retries.